cluster/plugin/local/plugin.go

```diff
@@ -248,9 +251,43 @@ func InfoNode(ctx context.Context, c *client.Client) (swarm.Node, error) {
 	return node, err
 }
 
-func InfoToJSON(sw swarm.Swarm, node swarm.Node) (string, error) {
+// InfoAMPCore returns the number of AMP core services
+func InfoAMPCore(ctx context.Context, c *client.Client) (int, error) {
+	var count int
+	services, err := c.ServiceList(ctx, types.ServiceListOptions{})
+	if err != nil {
+		return 0, err
+	}
+	for _, service := range services {
+		if strings.HasPrefix(service.Spec.Name, "amp") {
+			count++
+		}
+	}
+	return count, err
+}
+
+// InfoUser returns the number of user services
+func InfoUser(ctx context.Context, c *client.Client) (int, error) {
+	var count int
+	services, err := c.ServiceList(ctx, types.ServiceListOptions{})
+	if err != nil {
+		return 0, err
+	}
+	for _, service := range services {
+		if !strings.HasPrefix(service.Spec.Name, "amp") {
+			count++
+		}
+	}
+	return count, err
+}
+
+func InfoToJSON(sw swarm.Info, csCount int, usCount int) (string, error) {
```

Comment:
if we only use the swarm status, use a string parameter instead of the full swarm.Info structure